### ISSUE TITLE
Difficult to copy text out of scaladoc method description

### ISSUE THREAD

**Issue by NthPortal** [scala/bug#11223]
It's very difficult to copy text out of a scaladoc method description, because when you click to highlight the text, it collapses the method description.

**Comment by lrytz** [comment]
Related https://github.com/scala/scala/pull/6295

**Comment by Philippus** [comment]
@NthPortal Do you have a link where this is not working well right now?

**Comment by NthPortal** [comment]
oops, I forgot to come back to this and reply with the link.

The entry I had a problem with was [`BufferedIterator#takeWhile`](https://www.scala-lang.org/api/2.12.3/scala/collection/BufferedIterator.html#takeWhile(p:A=%3EBoolean):Iterator[A]), but I just realized that was 2.12.3. The equivalent entry for 2.12.7 works fine.

If only google would stop showing old ones

**Comment by NthPortal** [comment]
hijacking and re-opening this issue with a corner case where this is still problematic - copying the text out of the 'Full Signature' expandable sub-block on methods annotated with `@usecase`.

**Comment by SethTisue** [comment]
(`@usecase` is deprecated in 2.13, though)

**Comment by NthPortal** [comment]
@SethTisue true, but I don't think that means we should have poor behaviour until it's gone while people are potentially still using it. Also, IIRC Martin said he still thinks there's a lot of situations where it's really important/useful, so if/when it gets revived, it would be nice for it to behave correctly.

I agree that it's not super high priority though.

**Comment by NthPortal** [comment]
update: I'm not sure if anything has changed, but it's *doable*. you can highlight the 'Full Signature', which closes it. but if you click on 'Full Signature' again, it will reopen and the signature will still be highlighted, and you can copy it. it's certainly quite fiddly and a pain though
